.github/workflows/cla_check.yml

```diff
@@ -0,0 +1,34 @@
+name: CLA check
+
+# check if the PR author has signed the CLA
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+      - cla_check
+
+jobs:
+  cla_check_job:
+    runs-on:  ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-18.04]
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Install Dependecies
+        run: |
+          npm install googleapis@39
+          npm run actions-build
+          git add actions_build/index.js
+      - name: CLA check
+        uses: ./actions/cla_check
+        env:
+          SHEETS_TOKEN: ${{ secrets.SHEETS_TOKEN }}
+          SHEETS_CRED: ${{ secrets.SHEETS_CRED }}
+          GITHUB_TOKEN: ${{ secrets.TOKEN }}
+          SPREADSHEET_ID: ${{ secrets.SPREADSHEET_ID }}
```

Comment:
TODO: Remove file after testing is complete

Reply:
Can we remove it or is there some testing planned?

Reply:
A final test after @jameesjohn reviews it.